pydantic/fields.py

```diff
@@ -375,16 +379,18 @@ def Field(
       of digits within the decimal. It does not include a zero before the decimal point or trailing decimal zeroes.
     :param decimal_places: only applies to Decimals, requires the field to have at most a number of decimal places
       allowed. It does not include trailing decimal zeroes.
-    :param min_items: only applies to lists, requires the field to have a minimum number of
+    :param min_items: deprecated, use ``min_length`` instead.
```

Comment:
we want to use single back-ticks since this will be rendered as markdown - might as well fix the whole docstring now.

Reply:
Fixed for all docstrings in code